app/src/test/java/org/simple/clinic/newentry/form/DateOfBirthFieldTest.kt

```diff
@@ -2,11 +2,17 @@ package org.simple.clinic.newentry.form
 
 import com.google.common.truth.Truth.assertThat
 import org.junit.Test
+import org.simple.clinic.newentry.form.ValidationError.DateIsInFuture
 import org.simple.clinic.newentry.form.ValidationError.InvalidDateFormat
 import org.simple.clinic.newentry.form.ValidationError.MissingValue
+import org.threeten.bp.LocalDate
+import org.threeten.bp.format.DateTimeFormatter
 
 class DateOfBirthFieldTest {
-  private val dateOfBirthField = DateOfBirthField()
+  private val dateOfBirthField = DateOfBirthField(
+      DateTimeFormatter.ofPattern("dd/MM/yyyy"),
+      LocalDate.of(2019, 11, 20)
```

Comment:
I generally prefer to use `LocalDate.parse("2019-11-20")` since I feel the timestamp version is more human readable as a standard date format. There's a live template, `date`, in the plugin which should make adding this easier.

Reply:
Yes, that is how it looks like in one of the upcoming commits,
`LocalDate.parse("20/11/2019", dateTimeFormatter)`